numbers.js

```diff
@@ -4,7 +4,9 @@ function fibo(range){
     let f2=1
     let f3;
     let arr=[0,1]
-
+    if(range==0){
+        return undefined
+    }
     if(range==1){
         return 0
```

Comment:
your approach is good, but i see an issue here, when the range is other than 0 and 1 the return type would be array, but when when it is 1 the function should return 0 inside an array

Reply:
Fixed